src/lib/holocene/drawer.svelte

```diff
@@ -1,57 +1,55 @@
 <script lang="ts">
   import { fly } from 'svelte/transition';
 
+  import { setContext } from 'svelte';
+
   import { clickOutside } from '$lib/holocene/outside-click';
   import { focusTrap } from '$lib/utilities/focus-trap';
 
   import IconButton from './icon-button.svelte';
 
   export let open = false;
-  export let title: string;
   export let position: 'bottom' | 'right' = 'bottom';
   export let dark = true;
   export let onClick: () => void;
   export let id = 'navigation-drawer';
   export let closeButtonLabel: string;
+  export let closePadding: boolean = true;
```

Comment:
Is there a way to apply the padding based on an existing prop, like `position`?

Reply:
I couldn't think of a way to do that well without losing intent.

The issue is we need to be able to address the entire area of the drawer, and the padding we give content is the opposite of that.

<img width="586" alt="Screenshot 2024-01-30 at 9 55 03 AM" src="https://github.com/temporalio/ui/assets/1903839/16ab8948-08f3-44cc-bd86-25f0de086551">

The other ways to accomplish that is to give more access to the aside which felt worse as well so I settled on that prop to control it and default it because most times you won't need it.